cmd/go.mod

```diff
@@ -6,15 +6,15 @@ require (
 	github.com/filecoin-project/go-data-transfer v1.14.0
 	github.com/filecoin-project/go-legs v0.3.10
 	github.com/filecoin-project/index-provider v0.2.1
-	github.com/filecoin-project/storetheindex v0.3.8
+	github.com/filecoin-project/storetheindex v0.3.9-0.20220322103929-3e82a3cbc8ae
```

Comment:
Pending merge of the following PR for a tagged release: https://github.com/filecoin-project/storetheindex/pull/281